frontend/src/components/InfoWindow/InfoWindow.jsx

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 import DOMPurify from "dompurify";
+import { Button, BackTop } from "antd";
```

Comment:
I think we should stay consistent with the UI library we use throughout the app. Currently, Material UI is being used for everything else.

Reply:
@sultansidhu what do you think?

Reply:
Hmm I think we might wanna focus more on preserving the aesthetic of the overall project? For example, Antd has some gorgeous components that fit very well with material UI (even better than their material UI counterparts I'd argue). What do you guys think @slakh96 @mit1221

Reply:
I see. Ya, we can just leave it the way it is for now then. Maybe we can move everything to Antd later but that's definitely not a priority.